components/core/Text.tsx

```diff
@@ -44,3 +50,24 @@ export const LinkPrimaryVariant = styled(Link)`
   font-size: 16px;
   color: white;
 `;
+
+export function LinkifyText({ children }: LinkifyProps): ReactElement {
```

Comment:
Nitpick: It might be cleaner to use an inline type annotation (i.e. `{ children }: { children: ReactElement }`) instead of an one-off interface? Though I'm not sure if the linting will complain about it xD

Reply:
That works, but then Prettier pushes it into new lines

```typescript
export function LinkifyText({
  children,
}: {
  children: ReactElement;
}): ReactElement {
// ...
}
```

Reply:
Oh man, 5 lines for a function declaration :joy: Up to you on which to use!

Reply:
That's tough 😂 They're both not ideal LOL